guides/deployment/heroku.md

```diff
@@ -97,11 +97,11 @@ The buildpack uses a predefined Elixir and Erlang version, but to avoid surprise
 
 ```console
 # Elixir version
-elixir_version=1.14.0
+elixir_version=1.15.0
 
 # Erlang version
 # https://github.com/HashNuke/heroku-buildpack-elixir-otp-builds/blob/master/otp-versions
-erlang_version=24.3
+erlang_version=25.3
```

Comment:
btw I don't use Heroku, so cannot verify that this works

Reply:
We can go ahead and get feedback as required. But it should work, as the installation steps did not change for either.